.expeditor/verify.pipeline.yml

```diff
@@ -291,10 +246,6 @@ steps:
     executor:
       docker:
         image: ruby:2.7-buster
-        environment:
-          - FORCE_FFI_YAJL=ext
-          - CHEF_LICENSE=accept-no-persist
-          - UNIT_SPECS_27=1
```

Comment:
Makes sense overall but does this mean `UNIT_SPECS_27=1` and friends aren't doing anything?

Reply:
I'm fairly certain that's just a leftover from travis where we used the env vars to name the jobs.

Reply:
I grepped the project and could only find references to them in this file, so this seems 👍